### DISCUSSION TITLE
Need clarity on difference between multi-factor authentication to be able to authenticate and multi-factor authentication to be able to commit new code?

### DISCUSSION THREAD

**Discussion by wuwta-dev** [community/community#122611] [Code Security]
### Select Topic Area

General

### Body

We are asked the two questions in a security questionnaire:

a) Does your source code repository require multi-factor authentication to be able to authenticate?
Our Original answer: Yes, it has always been enabled.

b. Does your source code repository require multi-factor authentication to be able to commit new code?
I am unclear on what needs to be done for this question.
Can you please help me identify.

**Comment by azcguy** [comment]
As soon as you commit anything they will give you a month to enable MFA or you're locked out of committing anything.
They require your personal information, end goal is to fool you into giving Microsoft your information, none of this in reality requires MFA. 
Me personally, ill wait out this month, do something productive for other projects and then I'm out of here to other platforms.
